release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -2110,3 +2110,35 @@ link:https://access.redhat.com/solutions/6465271[{product-title} 4.9.5 container
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.9 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster by using the CLI] for instructions.
+
+[id="ocp-4-9-6"]
+=== RHBA-2021:4119 - {product-title} 4.9.6 bug fix and security update
+
+Issued: 2021-11-11
```

Comment:
This date needs to change to 2021-11-10. I will change after the input from peer review.